docs/build-and-publish-pipeline.md

```diff
@@ -21,14 +21,7 @@ The key points:
   tests via gradle.
 * [CircleCI](../.circleci/config.yml) runs on every branch, pull-request (including forks), and release,
   to execute lint checks and automated tests at the Rust and Swift level.
-* Releases are made by [manually creating a new release](./howtos/cut-a-new-release.md) via github,
-  which triggers various CI jobs:
-    * [CircleCI](../.circleci/config.yml) is used to build an iOS binary release on every release,
-      and publish it as GitHub release artifacts.
-    * [TaskCluster](../automation/taskcluster/README.md) is used to:
-        * Build an Android binary release.
-        * Upload Android library symbols to [Socorro](https://wiki.mozilla.org/Socorro).
```

Comment:
@bendk do you know if and where this happens? There is another reference to it but I'm not sure where we upload symbols?

Reply:
I didn't know about this and wouldn't be surprised if it wasn't happening.  @ahal does the new release-promotion code do this?